tests/commands/test_add.py

```diff
@@ -189,6 +190,7 @@ def test_add_set_cli(tmp_library: TemporaryLibrary) -> None:
     assert not doc.get_files()
 
 
+@pytest.mark.xfail(sys.platform == "win32", reason="Developer mode not enabled")
```

Comment:
Just curious, but can this be checked in any way? I'm thinking we could do a `pytest.skip` instead.